app/templates/suppliers/dashboard.html

```diff
@@ -1,5 +1,5 @@
 {% extends "_base_page.html" %}
-{% block page_title %}{{ current_user.supplier_name }} – Digital Marketplace{% endblock %}
+{% block page_title %}Supplier dashboard – Digital Marketplace{% endblock %}
```

Comment:
We don’t publicly call this a dashboard anywhere. In fact, we alternate between supplier name and ‘Your account’.

![image](https://cloud.githubusercontent.com/assets/355079/11777580/382f3be6-a247-11e5-9b63-07d652c36757.png)

![image](https://cloud.githubusercontent.com/assets/355079/11777572/2e7065ee-a247-11e5-8e38-3a03c0d60899.png)

--

‘Your account’ is definitely better than ‘Supplier dashboard’.